Updated SDK Version in `flutter.yaml` Workflow

milansurelia opened the pull request:
This PR will allow the workflow to run in the latest version.

wsloth commented:
I'm trying to get the build to work locally as well as in the pipeline, but not really getting very far. Are you able to run `flutter build apk` locally?

milansurelia commented:
Okay, will check & update soon

milansurelia commented:
@wsloth  I have updated the code.

Now, it's not breaking the CI/CD pipeline.
You can check this PR: https://github.com/wsloth/Index/pull/26


As I have updated the code in another [PR](https://github.com/wsloth/Index/pull/26
), I am closing this PR.
